### PR TITLE
Adding tko-vwt-dg 2.0

### PR DESCRIPTION
**Pull request by manishjha86**
Signed-off-by: Manish Jha <er.manish.jha@gmail.com>

### CODE REVIEW

File: src/deployment-guides/tko-on-vsphere-with-tanzu.md [hunk 1]
```diff
@@ -110,200 +105,216 @@ Do the following to deploy NSX Advanced Load Balancer Controller node:
...
 
    - IP Route Domain: Share IP route domain across tenants.
    - Service Engine Context: Service Engines are managed within the tenant context, not shared across tenants.
 
-   ![](./img/tko-on-vsphere-with-tanzu/image4.jpg)
+   ![Screenshot of Multi-Tenant screen](img/tko-on-vsphere-with-tanzu/TKO-VWT05.png)
+
+5. Click on **Save** to finish the post-deployment configuration wizard.
+
+   If you did not select the **Setup Cloud After** option <_I don't see this option_> before saving, the initial configuration wizard exits. The Cloud configuration window does not automatically launch and you are directed to a Dashboard view on the controller.
```
Comment (MarilynHorn) [inline src/deployment-guides/tko-on-vsphere-with-tanzu.md:143]:
```suggestion
   If you did not select the **Setup Cloud After** option before saving, the initial configuration wizard ends. The cloud configuration window does not automatically launch and you are directed to a dashboard view on the controller.
```

Comment (MarilynHorn) [reply src/deployment-guides/tko-on-vsphere-with-tanzu.md:143]: I removed my embedded comment **<_I don't see that option_>** from the MD file so I could explain it here, which is a better place for comments. When I reviewed this guide, I didn't see where the **Setup Cloud After** option is.  Was it shown in one of the screenshots you replaced, or did I just miss it? Please verify that the option is still in the wizard, and if it is, consider rewording this line so it's clearer.

**Review by MarilynHorn — CHANGES_REQUESTED** [review]
Overall, looks good. I removed one embedded comment at line 143 and added it as a comment here.

**Review by srinivasme21 — APPROVED** [review]
LGTM.

**Review by craigtracey — APPROVED** [review]

**Pull request merged**